axes/handlers/proxy.py

```diff
@@ -76,6 +76,7 @@ def update_request(request):
             request.axes_user_agent = get_client_user_agent(request)
             request.axes_path_info = get_client_path_info(request)
             request.axes_http_accept = get_client_http_accept(request)
+            request.axes_failures_since_start = 0
```

Comment:
This seems like an antipattern to set the value to 0? I'd much rather set it to None if it's not known, than set it to a value that can be misinterpreted?

Reply:
Makes sense. Fixed.